purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1541,11 +1543,11 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
      * Called immediately if purchaser info is cached. Purchaser info can be null if an error occurred.
      */
     @Suppress("DeprecatedCallableAddReplaceWith")
```

Comment:
ooo so this is how we could suppress the no "replace with"! if we wanted to for those others....

Reply:
Yes! I figured that out this morning

Reply:
what do you mean with "those others"? I think I did it for all of them, did I forget any?

Reply:
oh nope i must've missed those